stream.py

```diff
@@ -236,21 +235,34 @@ def stream_intro():
     p = subprocess.call(ffmpeg_args)
 
 def stream():
-    ffmpeg_stream = 'ffmpeg -thread_queue_size "%s" -f x11grab -draw_mouse 0 -s %s  -i :%d -thread_queue_size "%s" -f pulse -i default -ac 2 %s -f flv -flvflags no_duration_filesize "%s"' % (
-        args.ffmpeg_input_thread_queue_size, args.resolution, 122, args.ffmpeg_input_thread_queue_size, args.ffmpeg_stream_options, args.target)
-    logging.debug('Preparing to execute %r' % ffmpeg_stream)
-    ffmpeg_args = shlex.split(ffmpeg_stream)
-    logging.info("streaming meeting...")
-    p = subprocess.call(ffmpeg_args)
+    ffmpeg_outputs = []
+    if args.download:
```

Comment:
Thanks for your improovement. could you add a note, that the meeting is saved as file / streamed as it was in the original - so one can see in the logging, if download / streaming was active?

Reply:
Thanks for your review.
I have just added the same logging messages which were used before. Please take another look.